clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -198,6 +196,73 @@ public void testExists_NotExists() throws ApiException, IOException {
 
         Assert.assertFalse(fs.exists(p));
     }
+    
+    @Test
+    public void testDelete_FileExists() throws ApiException, IOException {
+        when(objectsApi.statObject(eq("repo"), eq("main"), eq("delete/sample/file.txt")))
```

Comment:
```suggestion
        when(objectsApi.statObject("repo", "main", "delete/sample/file.txt"))
```
(and similarly below)